ai-service/src/ai_service/db.py

```
@@ -42,4 +44,4 @@
             ids=ids,
         )
     except Exception as e:
-        raise Exception(f"Failed to add chunks to database: {e}") from e
+        raise DatabaseError(f"Failed to add chunks to database: {e}") from e
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)